### ISSUE TITLE
Image text linking works only when zone has @corresp and lb has @xml:id

### ISSUE THREAD

**Issue by ChiaraDipi** [evt-project/evt-viewer#109]
Image Text Linking is currently working properly only when there is a `<zone>` element connected to an `<lb>` element by a `@corresp` attribute pointing to the `@xml:id` of the lastest.
It would be very useful if it could work also with a connection between `<zone>` and `<lb>` that goes in the opposite direction, that is an `<lb>` connected to a `<zone>` element by a `@facs` attribute pointing to the `@xml:id` of the latest.

**Comment by mihalyeszter** [comment]
I agree! Transkribus gives TEI export this way. Is there any plan for such a development?

**Comment by RobertoRDT** [comment]
@mihalyeszter taking note of this for the next version (EVT 3).
